cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -1487,8 +1487,8 @@ function Install-Pigz {
 
 # Node Problem Detector Resources
 $NPD_SERVICE = "node-problem-detector"
-$DEFAULT_NPD_VERSION = '0.8.10-gke0.1'
-$DEFAULT_NPD_RELEASE_PATH = 'https://storage.googleapis.com/gke-release/winnode'
+$DEFAULT_NPD_VERSION = '0.8.20-0'
+$DEFAULT_NPD_RELEASE_PATH = 'https://storage.googleapis.com/gke-release'
```

Comment:
For reviewers it is in: https://console.cloud.google.com/storage/browser/gke-release/node-problem-detector
![54nNd3H3K5Cy3vD](https://github.com/user-attachments/assets/43eedb1e-9f37-44cc-abd8-2d3286a19455)

Reply:
Now I wonder if it should be: 0.8.20-0-g53e0152f-gke.0